src/extensions/combat/combatext_hooks.cpp

```diff
@@ -94,6 +96,50 @@ DECLARE_PATCH(_Explosion_Damage_IsWallAbsoluteDestroyer_Patch)
 }
 
 
+/**
+ *  #issue-897
+ *
+ *  Implements IsIceDestruction scenario option for preventing destruction of ice,
+ *  and IceStrength for configuring the chance for ice getting destroyed.
+ *
+ *  @author: Rampastring
+ */
+DECLARE_PATCH(_Explosion_Damage_IsIceDestruction_Patch)
+{
+    GET_REGISTER_STATIC(const WarheadTypeClass*, warhead, edi);
```

Comment:
Needs a space between `WarheadTypeClass` and `*`.

This is most likely your MSVC auto-formatting taking over here, you can change those easily in Tools -> Options -> Text Editor -> C/C++.